### PR TITLE
add share files and simple paste site

### PR DESCRIPTION
**Pull request by DIp15739**
## Requirements

 * [x] This is Software as a Service not self hosted
 * [x] It has a free tier not just a free trial
 * [x] Pricing information is clearly visible without signup or phone calls
 * [x] The submission mentions what is free
 * [x] The submission is not already present in the list
 * [x] The service has contact details of those running it and a privacy policy

### CODE REVIEW

**Comment by nextpier** [comment]
Broken link...

File: README.md [hunk 1]
```diff
@@ -471,6 +471,8 @@ This list results from Pull Requests, reviews, ideas, and work done by 1100+ peo
   * [Webex](https://www.webex.com/) — Video meetings with a free plan offering 40 minutes per meeting with 100 attendees.
   * [RingCentral](https://www.ringcentral.com/) — Video meetings with a free plan offering 50 minutes per meeting with 100 participants.
   * [GitBook](https://www.gitbook.com/) — Platform for capturing and documenting technical knowledge — from product docs to internal knowledge bases and APIs. Free plan for individual developers.
+  * [transfernow](https://www.transfernow.ne) — simplest, fastest and safest interface to transfer and share files. Send photos, videos and other large files without a manditory subscription.
```
Comment (ripienaar) [inline README.md:474]: indeed, missing `t` in the link

Comment (DIp15739) [reply README.md]: fix and push

**Comment by ripienaar** [comment]
Thanks

**Pull request merged**